diff -Nur trunk/infobash trunk.new/infobash
--- trunk/infobash	2006-11-29 14:42:20.000000000 +0100
+++ trunk.new/infobash	2006-11-29 18:39:26.000000000 +0100
@@ -403,7 +403,7 @@
 ##say "ISSUE='$ISSUE'"
 case $(echo "$ISSUE" | tr [:upper:] [:lower:]) in
   *debian*)
-    for I in /etc/kanotix-version /etc/knoppix-version; do
+    for I in /etc/sidux-version /etc/kanotix-version /etc/knoppix-version; do
       [[ -s $I ]] && DISTRO=$(eat $I) && break
     done
     if [[ -z $DISTRO ]]; then 
